daprdocs/content/en/developing-applications/building-blocks/state-management/state-management-overview.md

```diff
@@ -104,37 +130,52 @@ KEYS "myApp*"
 ```
 
 {{% alert title="Note on direct queries" color="primary" %}}
-Direct queries of the state store are not governed by Dapr concurrency control, since you are not calling through the Dapr runtime. What you see are snapshots of committed data which are acceptable for read-only queries across multiple actors, however writes should be done via the Dapr state management or actors APIs.
+Since you aren't calling through the Dapr runtime, direct queries of the state store are not governed by Dapr concurrency control. What you see are snapshots of committed data acceptable for read-only queries across multiple actors. Writes should be done via the Dapr state management or actors APIs.
 {{% /alert %}}
 
 ##### Querying actor state
-If the data store supports SQL queries, you can query an actor's state using SQL queries. For example use:
+
+If the data store supports SQL queries, you can query an actor's state using SQL queries. For example:
 
 ```sql
 SELECT * FROM StateTable WHERE Id='<app-id>||<actor-type>||<actor-id>||<key>'
 ```
 
-You can also perform aggregate queries across actor instances, avoiding the common turn-based concurrency limitations of actor frameworks. For example, to calculate the average temperature of all thermometer actors, use:
+You can also avoid the common turn-based concurrency limitations of actor frameworks by performing aggregate queries across actor instances. For example, to calculate the average temperature of all thermometer actors, use:
 
 ```sql
 SELECT AVG(value) FROM StateTable WHERE Id LIKE '<app-id>||<thermometer>||*||temperature'
 ```
 
 ### State Time-to-Live (TTL)
+
 Dapr enables per state set request time-to-live (TTL). This means that applications can set time-to-live per state stored, and these states cannot be retrieved after expiration.
```

Comment:
Can you add a link here to this article https://v1-8.docs.dapr.io/developing-applications/building-blocks/state-management/state-store-ttl/. 
Generally all the sub-articles should be referenced from the overview page.